framework_tvb/tvb/interfaces/rest/client/examples/fire_simulation.py

```diff
@@ -118,11 +118,32 @@ def fire_simulation_example(tvb_client_instance):
         assert chunk.shape[2] == data_shape[2]
         assert chunk.shape[3] == data_shape[3]
 
+        return project_gid, time_series_gid
+
+
+def quick_launch_an_operation(tvb_client_instance, project_gid, time_series_gid):
```

Comment:
Maybe rename `time_series_gid ` into something a bit more generic?
It could be any `datatype_gid`, right?

Reply:
yes, sure